vllm/inputs/registry.py

```diff
@@ -244,6 +244,10 @@ def dummy_data_for_profiling(
         if dummy_data.multi_modal_data is not None:
             for k, v in dummy_data.multi_modal_data.items():
                 num_items = len(v) if isinstance(v, list) else 1
+                #fix for molmo 
+                if k == "image" and model_config.hf_config.model_type == "molmo":
```

Comment:
In any case, let's not hard code this here. Can you instead modify the dummy data inside the model's file?

Reply:
ok